### PR TITLE
feat: feedback skeleton ui

### PR DESCRIPTION
**Pull request by sjaanus**
This adds component, which will allow to draw feedback component based on incoming data.

### CODE REVIEW

File: frontend/src/component/feedbackNew/FeedbackProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+interface IFeedbackProviderProps {
+    children: ReactNode;
+}
+
+export const DEFAULT_FEEDBACK_DATA = {
+    category: 'general',
+};
+export const FeedbackProvider = ({ children }: IFeedbackProviderProps) => {
+    const [feedbackData, setFeedbackData] = useState<ProvideFeedbackSchema>(DEFAULT_FEEDBACK_DATA);
```
Comment (sjaanus) [inline frontend/src/component/feedbackNew/FeedbackProvider.tsx:14]: This type will change in future, and take in just feedback category and user type.

File: frontend/src/component/feedbackNew/FeedbackComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+export const StyledContainer = styled('div')(({ theme }) => ({
+    position: 'fixed',
+    top: 0,
+    right: 0,
+    width: '496px',
+    height: '100vh',
+    opacity: 1,
+    backgroundColor: theme.palette.primary.main,
+    zIndex: 300,
```
Comment (kwasniew) [inline frontend/src/component/feedbackNew/FeedbackComponent.tsx:13]: we have design system for zIndex elements available on theme

File: frontend/src/component/feedbackNew/FeedbackComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
+import { Box, styled } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { useFeedback } from './useFeedback';
+
+export const StyledContainer = styled('div')(({ theme }) => ({
+    position: 'fixed',
+    top: 0,
+    right: 0,
+    width: '496px',
```
Comment (kwasniew) [inline frontend/src/component/feedbackNew/FeedbackComponent.tsx:9]: does it look good on mobile?

File: frontend/src/component/feedbackNew/FeedbackProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    );
+
+    const [showFeedback, setShowFeedback] = useState(false);
+    const openFeedback = (data: ProvideFeedbackSchema) => {
+        setFeedbackData(data);
+        setShowFeedback(true);
+    };
+
+    const closeFeedback = () => {
+        setFeedbackData(DEFAULT_FEEDBACK_DATA);
```
Comment (kwasniew) [inline frontend/src/component/feedbackNew/FeedbackProvider.tsx:25]: what is the default value use for when the feedback is hidden?

File: frontend/src/component/feedbackNew/FeedbackProvider.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import { FeedbackComponent } from './FeedbackComponent';
+import { FeedbackContext } from './FeedbackContext';
+import { ReactNode, useState } from 'react';
+import { ProvideFeedbackSchema } from '../../openapi';
+
+interface IFeedbackProviderProps {
+    children: ReactNode;
```
Comment (kwasniew) [inline frontend/src/component/feedbackNew/FeedbackProvider.tsx:7]: I think you can skip it if you type your component as FC since it's a React type

**Review by kwasniew — APPROVED** [review]

**Pull request merged**